src/components/PopularVariants.js

```diff
@@ -6,7 +6,9 @@ function PopularVariant({ src, name, description }) {
   return (
     <div>
       <h3>{name}</h3>
-      <Image src={src} alt="Variant" width={400} height={400} />
+      <a href="/variants">
```

Comment:
Use \<Link\> from react-router-dom instead of \<a\>. \<Link\> would just render the page component while \<a\> will reload the entire page

Reply:
I changed it but its giving this error:
![image](https://user-images.githubusercontent.com/45343196/68732630-201f8980-05a2-11ea-9da1-3f20782b5d36.png)
Same thing happens when clicking on "Explore Variants" in the Homepage Nav bar.